com.unity.ml-agents/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Major Changes
 #### com.unity.ml-agents (C#)
 #### ml-agents / ml-agents-envs / gym-unity (Python)
+ - TensorFlow trainers have been deprecated, please use the Torch trainers instead. (#4707)
```

Comment:
```suggestion
 - TensorFlow trainers have been removed. All trainers now use the Torch. (#4707)
```
"Deprecated" implies it's still there. But there's no longer a choice.

We should probably put something in the migration guide about onnx files too.